libs/core/langchain_core/language_models/chat_models.py

```diff
@@ -366,7 +366,7 @@ class BaseChatModel(BaseLanguageModel[AIMessage], ABC):
 
     @model_validator(mode="before")
     @classmethod
-    def raise_deprecation(cls, values: dict) -> Any:
+    def raise_deprecation(cls, values: typing.Dict) -> Any:  # noqa: UP006
```

Comment:
We're dropping 3.9.
```suggestion
    def raise_deprecation(cls, values: dict) -> Any:
```
will this work?